Code/Framework/AzFramework/AzFramework/Entity/EntityDebugDisplayBus.h

```diff
@@ -33,6 +33,156 @@ namespace AzFramework
 {
     inline constexpr AZ::s32 g_defaultSceneEntityDebugDisplayId = AZ_CRC_CE("MainViewportEntityDebugDisplayId"); // default id to draw to all viewports in the default scene
 
+    // Notes:
+    //   Don't change the xxxShift values, they need to match the values from legacy cry rendering
+    //   This also applies to the individual flags in EAuxGeomPublicRenderflags_*!
+    // Remarks:
+    //   Bits 0 - 22 are currently reserved for prim type and per draw call render parameters (point size, etc.)
+    //   Check RenderAuxGeom.h in ../RenderDll/Common
+    enum EAuxGeomPublicRenderflagBitMasks
```

Comment:
Might be a good idea to add a enum type specifier here 
```suggestion
    enum EAuxGeomPublicRenderflagBitMasks : AZ::u32
```